fgspectra/cross.py

```diff
@@ -112,9 +112,17 @@ def __call__(self, sed_args, cl_args):
         cross : ndarray
             Cross-spectrum. The shape is ``(..., freq, freq, ell)``.
         """
-        f_nu = self._sed(*sed_args)
+
+        f_nu1, f_nu2 = self._sed(*sed_args)
+        cl = self._cl(*cl_args)
+        for _ in range(3-cl.ndim):
+            cl = cl[np.newaxis, ...]
```

Comment:
I don't understand the frequency unpacking. Is the goal evaluating  nu1 x nu2 instead of the whole block ((11, 12), (21, 22))?